sass/_SimpleFilter.scss

```diff
@@ -77,6 +78,11 @@
   height: 20px;
 }
 
+.coveo-simplefilter-toggle-svg-container {
+  @include display(flex);
+  padding-top: 2px;
```

Comment:
Since we have `display:flex` set, would `align-items: stretch` work instead of using a arbitrary number to correct the alignment?

Reply:
I tried it, and it's the same issue. While "aligned" for real in the document, it just does not look right visually since the middle of the text + middle of the arrow give the sense that it's mis-aligned.